Dockerfile

```diff
@@ -3,17 +3,22 @@ FROM ubuntu:18.04
 LABEL ubuntu.version="18.04"
 MAINTAINER Tom Eichlersmith <eichl008@umn.edu>
 
+ARG NPROC=1
```

Comment:
I like this solution :raised_hands:

Reply:
I've only tested locally with setting the argument (works), but I imagine the runner will do the default case.